### DISCUSSION TITLE
Sidebar slider step should probably be applied to the input

### DISCUSSION THREAD

**Discussion by positlabs** [cables-gl/cables_docs#253] [Feature Requests]
I feel like the step behavior should be applied to the input value, not just the slider value change. You can see in this patch, I'm using step .01 but it's only applied if I use the slider. The input value is unchanged, which leads to long values like 0.023124124 which I don't want. 

![Screen Shot 2022-05-28 at 2 24 54 PM](https://user-images.githubusercontent.com/1918844/170838314-b7cf7081-ed65-4172-8343-2888af30479d.png)

https://cables.gl/edit/6kk1nZ

**Comment by steam0r** [comment]
hey, thanks for the suggestions, this is kind of on purpose as we are not doing any rounding of input values here (as i think you suggest), you can always round your values using math-ops before putting them into the sidebar-input-value.

**Reply by positlabs** [reply]
Yeah I could do that, but in general I expect the value options would be applied to the op output, regardless of where the raw value is coming from. What's the reasoning for only applying the step to the slider? Seems like it could lead to unexpected behavior.
